changelog/_unreleased/2025-03-27-meta-information-in-landing-pages-is-not-inherited.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Meta information in landing pages is not inherited
+issue: https://github.com/shopware/shopware/issues/3712
+author: Stefan Reichelt
+author_email: stefan@kreativsoehne.de
+author_github: @Songworks
+---
+# Storefront
+* Changed `LandingPageLoader` to retrieve meta information (metaTitle, metaDescription & keywords) through translations, for proper language inheritance
+* Deprecates `PageNotFoundException` in `LandingPageLoader` for v6.8.0, in favour of `LandingPageException::notFound`
```

Comment:
```suggestion
* Deprecated `PageNotFoundException` in `LandingPageLoader` for v6.8.0, in favour of `LandingPageException::notFound`
```

Was about to push this myself to this MR to accelerate things, but it seems, you didn't give permissions to let us push to your fork 🤔

So for this time: Please change this and we can merge it 😄